mod_reforged/hooks_helper.nut

```diff
@@ -0,0 +1,82 @@
+::Reforged.HooksHelper <- {
+	function dualWeapon( q )
+	{
+		q.m.Weapon1 <- null;
+		q.m.Weapon2 <- null;
```

Comment:
There should be a comment here describing what these variables need to contain in order for this weapon to work properly.
> Strings? Instanitated Weapons? Can they be empty?